airbyte-integrations/connectors/source-convex/integration_tests/sample_config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "deployment_name": "descriptive-vulture-260",
+  "access_key": "Your access token"
```

Comment:
how do i give the access key used in acceptance tests? it's stored in `secrets/config.json` locally which is gitignored.